gammapy/estimators/points/tests/test_core.py

```diff
@@ -336,3 +337,22 @@ def test_flux_points_plot_no_error_bar():
     flux_points = FluxPoints.from_table(table)
     with mpl_plot_check():
         _ = flux_points.plot(sed_type="dnde")
+
+
+def test_fp_no_is_ul():
+    dataset_1 = simulate_map_dataset(random_state=0, name="dataset_1")
```

Comment:
It might be better not to import and use here the estimator. I would rather suggest to import a `FluxPoints` from a table where you remove the UL column.
See e.g. `test_plot_fp_no_ul`